doc/designs/ossl-provider-load-ex.md

```diff
@@ -1,12 +1,12 @@
-Providers run-time configuration
-================================
+OSSL_PROVIDER_load_ex - activating providers with run-time configuration
+========================================================================
```

Comment:
Not sure. The document originally was intended to describe all the possible options, not only the new one.

Reply:
I don't quite follow. This here is just a design document, not the official documentation or a manual page. You proposed a new API, `OSSL_PROVIDER_load_ex` and this is the design document for it. Mentioning the alternatives serves as a rationale for your design decisions but it is not the goal of this document to document all existing options. I would expect those to be already documented in detail in our manpages ;-)

Reply:
Fair point, I agree, thanks!